nixops/script_defs.py

```diff
@@ -777,15 +777,14 @@ def op_ssh(args):
         if not m:
             raise Exception("unknown machine ‘{0}’".format(machine))
         flags, command = m.ssh.split_openssh_args(args.args)
-        user = None if username == "root" else username
```

Comment:
I don't quite follow what is going on here

Reply:
This is argument handling for `nixops ssh ...`